README.md

```diff
@@ -19,10 +19,16 @@ The project is written in Java 8.
 ```xml
    <dependency>
        <groupId>pl.edu.icm.spark-utils</groupId>
-       <artifactId>spark-utils</artifactId>
-       <version>1.0.0</version>
+       <artifactId>spark-utils_2.10</artifactId>
+       <version>1.0.1</version>
+   </dependency>
+   <dependency>
```

Comment:
Let's separate those two dependency definitions as alternative options (into two alternative code snippets). Just to make sure people reading this description would not blindly copy-paste both dependencies while only one is required.